.gitignore

```diff
@@ -316,3 +316,6 @@ platform/windows/godot_res.res
 
 # Scons progress indicator
 .scons_node_count
+
+# GodotSharp nuget packages
+modules/mono/editor/GodotSharpTools/packages
```

Comment:
It would be better to create `modules/mono/editor/GodotSharpTools/.gitignore` and add this there.

Reply:
Good idea!